tests/terraform/checks/test_multiple_skips.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+from pathlib import Path
+
+from checkov.runner_filter import RunnerFilter
+from checkov.terraform.runner import Runner
+
+
+class TestMultipleSkips(unittest.TestCase):
+
+    def test(self) -> None:
+        # given
+        test_files_dir = Path(__file__).parent / "a example skip"
```

Comment:
```suggestion
        test_files_dir = Path(__file__).parent / "a_example_skip"
```
Please use one word in the file names.